index.html

```diff
@@ -2421,7 +2422,7 @@ <h2 id="Sync">Backbone.sync</h2>
     </p>
 
     <ul>
-      <li><b>method</b> – the CRUD method (<tt>"create"</tt>, <tt>"read"</tt>, <tt>"update"</tt>, or <tt>"delete"</tt>)</li>
+      <li><b>method</b> – the CRUD method (<tt>"create"</tt>, <tt>"read"</tt>, <tt>"update"</tt>, <tt>"patch"</tt> or <tt>"delete"</tt>)</li>
```

Comment:
Updated the docs here to include that `patch` can also be passed into `Backbone.Sync` under the `method` parameter.